### PR TITLE
feat: add prisma for DB & migrations

### PR DESCRIPTION
**Pull request by smessie**

### CODE REVIEW

File: server/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+volumes:
+  pgdata-zeusssss:
+
+services:
+  db:
+    image: postgres:12-alpine
```
Comment (NuttyShrimp) [inline server/docker-compose.yml]: bump version to an updated version

**Comment by NuttyShrimp** [comment]
We don't remove other persons work

File: server/src/index.ts [hunk 2]
```diff
@@ -13,6 +16,11 @@ server.get("/", (_, res) => {
   res.sendFile("index.html");
 });
 
+server.get("/teams", async (request, reply) => {
```
Comment (NuttyShrimp) [inline server/src/index.ts]: dit moet onder een `/api` router/path komen

**Review by NuttyShrimp — CHANGES_REQUESTED** [review]

**Review by hanneskt — APPROVED** [review]
Database looks good! Didn't check the rest. Love from @FKD13 <3

**Review by NuttyShrimp — APPROVED** [review]

**Pull request merged**